config/application.rb

```diff
@@ -343,6 +344,30 @@ def init_dynflow
     def setup_auditing
       Audit.send(:include, AuditSearch)
     end
+
+    def notify_deprecations
+      if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?('mysql')
+        blueprint = NotificationBlueprint.find_by_name('feature_deprecation')
+        Foreman::Deprecation.deprecation_warning('1.24', "Support for MySQL is deprecated, migrate to PostgreSQL")
```

Comment:
maybe consider adding "please migrate" to the text (or make it sound less like an order :)

Reply:
Good idea.